package.json

```diff
@@ -39,7 +39,7 @@
     "build:main": "npx tsc -p tsconfig.json && npx ncp package.json ./build/lib/package.json && npx ncp README.md ./build/lib/README.md && npx ncp .npmignore ./build/lib/.npmignore",
     "build:module": "npx tsc -p tsconfig.module.json",
     "build:web": "npx tsc -p tsconfig.json --module amd --out build/lib/machinelearn.web.js",
-    "lint": "npx tslint -t verbose 'test/**/*.ts' 'src/**/*.ts' 'docs/**/*.ts' && npx prettier \"src/**/*.ts\" \"test/**/*.ts\" \"docs/**/*.ts\" --config ./.prettierrc.json --list-different",
+    "lint": "npx tslint -t verbose 'test/**/*.ts' 'src/**/*.ts' 'docs/**/*.ts' && pretty-quick --staged",
```

Comment:
if we are going to choose this direction, is it possible to run tslint:fix on the staged files as well? Then we can remove `npx tslint -t verbose 'test/**/*.ts' 'src/**/*.ts' 'docs/**/*.ts'`